src/pages/home/report/ReportActionItem.js

```diff
@@ -47,13 +47,14 @@ const propTypes = {
     index: PropTypes.number.isRequired,
 
     /** Draft message - if this is set the comment is in 'edit' mode */
-    draftMessage: PropTypes.string,
+    // eslint-disable-next-line react/no-unused-prop-types
+    reportActionsDrafts_: PropTypes.objectOf(PropTypes.string),
```

Comment:
The comment description would have to change if now the prop here receives all available draft messages